randovania/layout/base/all_obtainable_option.py

```diff
@@ -0,0 +1,22 @@
+from enum import Enum
+
+from randovania.bitpacking.bitpacking import BitPackEnum
+from randovania.lib import enum_lib
+
+
+class AllObtainableOption(BitPackEnum, Enum):
+    long_name: str
+
+    DISABLED = "disabled"
+    MAJORS = "majors"
+    ALL = "pickups"
```

Comment:
I think a better name for this setting would be "Logical Pickup Placement", with the options "Minimal", "Major pickups", "All pickups". This class in particular can then be renamed to `LogicalPickupPlacementConfiguration`